app/models/manageiq/providers/microsoft/infra_manager/powershell.rb

```diff
@@ -9,15 +9,16 @@ def execute_powershell(connection, script)
 
       def run_powershell_script(connection, script)
         log_header = "MIQ(#{self.class.name}.#{__method__})"
-        File.open(script, "r") do |file|
-          begin
-            results = connection.run_powershell_script(file)
-            log_dos_error_results(results)
-            results
-          rescue Errno::ECONNREFUSED => err
-            $scvmm_log.error "MIQ(#{log_header} Unable to connect to SCVMM. #{err.message})"
-            raise
+        begin
+          file = IO.read(script)
```

Comment:
Not sure what the change from File.open to IO.read has to do with the purpose of this PR.  The rest of it seems good.